oio/cli/container/container.py

```diff
@@ -253,38 +299,41 @@ def get_parser(self, prog_name):
 
     def take_action(self, parsed_args):
         self.log.debug('take_action(%s)', parsed_args)
-
+        super(FlushContainer, self).take_action_container(parsed_args)
+        if parsed_args.cid is not None:
+            data = self.app.client_manager.storage.container_get_properties(
+                self.app.client_manager.account,
+                parsed_args.container,
+                cid=parsed_args.cid
+            )
+            parsed_args.container = data['system']['sys.user.name']
```

Comment:
This pattern (getting container name from a CID) is repeated several times. It deserves a dedicated method, like `resolve_cid`, which would return an account and container name (as a tuple or a dict), by requesting meta1 (here we are requesting meta2). This could be done in another PR, though.